Fix of Educational Site

sebastian-meier opened the pull request:
The content shows up twice. Remove the Content from the site (as its already included in the BaseLayout.

See also current version here: https://p5js.org/education-resources/

sebastian-meier committed:
Fix of Educational Site

The content shows up twice. Remove the Content from the site (as its already included in the BaseLayout.

sebastian-meier committed:
Adding parametric-design@fhp to educational ressources

davepagurek commented on src/pages/education-resources/index.astro:
```diff
@@ -39,5 +39,4 @@ setJumpToState({
   variant="item"
   topic="tutorials"
 >
-  <Content />
```
Thanks for the fix! Since we're no longer using this, I think we can delete these lines too, which I believe are only used for creating the `Content` component used here:
https://github.com/processing/p5.js-website/blob/6a83470e7cf64bc30d845f19c8bf01089d559188/src/pages/education-resources/index.astro#L10-L13

sebastian-meier replied:
I removed the last line. The other two are actually requried.

sebastian-meier committed:
Merge branch 'main' into main

sebastian-meier committed:
Merge branch 'processing:main' into main

sebastian-meier committed:
Update index.astro

Removing obsolete lines according to conversation on pull request: https://github.com/processing/p5.js-website/pull/574

sebastian-meier committed:
Update index.astro

Two of the three lines were actually required. And cannot be removed.

sebastian-meier committed:
Update index.astro

Fixed the removal of code (see pull request)
